frontend/src/data/headerLinks.ts

```diff
@@ -1,20 +1,16 @@
 import { IHeaderLink } from './interfaces'
 
 export const headerLinks: IHeaderLink[] = [
+  {
+    name: 'Dashboard',
+    href: '/dashboard'
+  },
   {
     name: 'Categories',
     href: '/categories'
   },
   {
     name: 'User List',
     href: '/user-list'
-  },
-  {
-    name: 'Profile',
-    href: '/profile'
-  },
-  {
-    name: 'Settings',
-    href: '/settings'
```

Comment:
why where these removed? Are these not needed for navigation?

Reply:
This is the reason why I remove those other links.

I just created new dropdown and show it in the other component

![image](https://user-images.githubusercontent.com/108642414/185293846-59e8c783-b8a0-4b3d-b540-355a513f64bd.png)